### ISSUE TITLE
Solutioning for high-priority Design Intent challenges

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#56730]
## User Story
As a Governance team member, I want to spend some time researching, analyzing, and coming up with ideas to resolve the top, high-priority challenges we face at Design Intent so that my team can introduce changes to our process to better handle and/or mitigate these challenges.

Assignee: @allison0034 
Peer Reviewer: @sterkenburgsara 

## Description
In ticket #55646, Governance team completed an asynch workshop focusing on the challenges and pain points faced at the Design Intent touchpoint. The top 2 challenges Governance team faces are:
1. VFS teams coming to Design Intent with high-fidelity designs, opposed to low-fi designs, wireframes and/or sketches.
2. Assumption: VFS teams are not "bought in" to the Design Intent... aka they don't see it's value. 
(Note: We also recognized that many teams are not scheduling their DI on the timeline preferred by the Governance team. They are scheduling them too late, when they've already made significant contributions to their proposed design. This could be the reason why we see so many high-fidelity designs, and it also could be the reason why teams aren't "bought in" to the DI process. Something to consider.)

The purpose of this ticket is to dig into the above challenges a bit deeper and propose solutions for how to best handle them. Questions we may want to ask ourselves wrt the solution:

- Is a solution necessary? Are these problems significant enough where changing/modifying our process is warranted?
- Should we accept the problem as-is and modify our operational process to better handle the level of prototypes that are shared with us?
- How can we encourage VFS teams to engage with us earlier?
- Is there a way to mandate that teams must engage with us for a DI within so many days post kick-off?
- Could we provide examples of what we're looking for? Would that be helpful?
- What about testimonials from teams who have completed the DI/Collaboration Cycle process?
- Is it just a matter of changing the language on our Design Intent touchpoint page?
- Is there 1 solution that could help us solve both problems or does this require multiple changes?

## Impacted Artifacts
TBD

## Tasks
- [ ] Document proposed solutions in a Confluence document on the Governance team page
- [ ] Share proposed changes with @humancompanion-usds for review and feedback.

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] We have identified a solution for addressing the high-fidelity prototypes shared at Design Intent
- [ ] We have identified a solution for getting VFS teams "bought in" to the Design Intent

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by humancompanion-usds** [comment]
> VFS teams are not "bought in" to the Design Intent... aka they don't see it's value.

Is this based on research with VFS teams or is it a hypothesis we have? If we've got research, can you point me to it? If we don't then I'd like to engage the Service Design team to dig into this with VFS teams. We shouldn't have to guess. We can ask them.

**Comment by shiragoodman** [comment]
This is not based on research. This was something that came out of our workshop in our last sprint. I think it's better classified as an assumption.

We have not done a lot of research with VFS teams on the Collaboration Cycle. In the past when we have done research, it's been mostly usability testing of new changes/processes we're piloting. Service Design _may _ have findings from their Platform Satisfaction Survey that could support this claim, but I don't know for certain.

I am supportive of getting Service Design involved. I think because our team is so embedded in the Collaboration Cycle process, it would be challenging for us to objectively take on research without having our own biased opinions.

**Comment by humancompanion-usds** [comment]
Thanks, @shiragoodman, that's helpful context. I'd like to find a time to go over the workshop findings around design intent specifically with the team. I don't want to discourage you all from fixing problems in the process, but I also want to understand a bit better what is driving those changes, what we hope to achieve, and how we'll measure that.

I understand that Service Design currently uses surveys to keep tabs on how things are going but I'm going to be working with them to do more qualitative research sessions with teams. So let's talk about things we want to find out from teams. 

Also, as an aside, Charles shared this slide in his presentation to USDS on the VA.gov history yesterday:
<img width="1034" alt="Screen Shot 2023-04-12 at 4 35 58 PM" src="https://user-images.githubusercontent.com/52176351/231578352-cb2bbe0a-f1c3-4570-9686-a90bb98ce261.png">

The point was that there are some axioms of Civic Tech (and design in general) that we dogmatically adhere to that may in fact be correct to occasionally violate. The story is that it actually took making a high-fidelity mock-up of a reimagined va.gov to get buy-in to build it (and we repeated that for the mobile app). Not exactly the same thing as a team coming to design intent, clearly. But my point is that perhaps it's worth considering and defining when it is okay to deviate from some of these rules and expectations.

**Comment by allison0034** [comment]
We decided in the meeting to work with Service Design team to get a better understanding of why VFS teams come with high fi comps. Matt is going to speak to Brian Smolensky and then I will meet with him when I am back from OoO to come up with a research plan.

**Comment by sterkenburgsara** [comment]
Update: Allison is OOO and we are delayed until she can schedule relevant meetings when back in the office. Thus, this ticket will move into the next sprint.

**Comment by allison0034** [comment]
I have a meeting set up with Brian on Monday 5/8/23 to discuss.

**Comment by allison0034** [comment]
@shiragoodman lol doh! Sorry I had his last name in the comment above that one.

**Comment by shiragoodman** [comment]
https://dsva.slack.com/archives/GNF5GSD9N/p1683640126206739

Slack conversation with @humancompanion-usds above. Waiting on decision regarding receiving support from Service Design. This ticket will remain open until a path forward has been defined and approved by PO.

**Comment by shiragoodman** [comment]
@allison0034 this ticket is good to close. Can you create a new ticket on our short-term solution for having a "Design Intent Info session" ?

**Comment by allison0034** [comment]
New [ticket](Design Intent Info session at Veteran Digital User Experience Weekly Design Sync) has been created.

**Comment by shiragoodman** [comment]
@allison0034 can you update your comment above? Doesn't look like it linked correctly.
